README.md

```diff
@@ -8,6 +8,8 @@
 
 **duo_python** - Duo two-factor authentication for Python web applications: https://duo.com/docs/duoweb
 
+Checkout our new [duo_universal_python](https://github.com/duosecurity/duo_universal_python) repo.
```

Comment:
Could you add that this repo will eventually be deprecated in favor of the new one?
I think it will be good to be slightly more aggressive in our marketing here :)

Reply:
@xdesai While that's true, at this time we can't say when that will happen, or give them an alternative option.  So right now I don't see that as compelling or useful information until we have more specific.

Reply:
that's fine with me. I think Briar's latest update at least clarifies a little more why someone would want to switch so that's good enough for me.